lib/pugin/view_helpers.rb

```diff
@@ -8,4 +8,52 @@ def navigation_link(route_symbol:, letter:, primary_id: nil, secondary_id: nil)
 			return link_to(letter, url, data: { 'atoz-see': 'true' })
 		end
 	end
+	module Feature
+		module Status_banner
```

Comment:
They get called in the host apps via methods like 
def enable_navigation_main
    @enable_top_navigation = Pugin::Feature::Top_navigation.enable
  end

This seems like a sketchy implementation however and should be re-looked at.